### ISSUE TITLE
stripe integration

### ISSUE THREAD

**Issue by chiamtc** [weikee94/node-with-react#1]
This is so fun to implement with node together. Stripe API docs is so good ! 💪

**Comment by weikee94** [comment]
yeah man, currently i trying to learn how to integrate both node with react 
hopefully the front+back small example can release soon and give u a try

**Comment by chiamtc** [comment]
stripe API is really good - my team and I implmented it for a subscription-based model feature. We use google cloud function - like an endpoint, use stripe secret key to query to stripe. In this project, I think you could write API endpoints query to stripe and expose it for frontend to consume.

**Comment by chiamtc** [comment]
Is this also a course?

**Comment by weikee94** [comment]
@chiamtc yes another from udemy [https://www.udemy.com/node-with-react-fullstack-web-development/learn/v4/t/lecture/7603030?start=0](url) 

havent got the chance to try the stripe btw you also implement with node as backend as well ?

**Comment by chiamtc** [comment]
fb
